.github/workflows/release.yml

```
@@ -25,12 +24,9 @@ jobs:
             - name: Merge latest-mac.yml
               run: |
                   ls -R
-                  rm package.json package-lock.json
                   mv nrfconnect-macOS-x64/latest-mac-x64.yml nrfconnect-macOS-arm64/
-                  mv scripts/mergeLatestYmls.js nrfconnect-macOS-arm64/
                   cd nrfconnect-macOS-arm64
-                  npm i js-yaml
-                  node mergeLatestYmls.js latest-mac-x64.yml latest-mac-arm64.yml latest-mac.yml
+                  yq eval-all '(select(fileIndex == 0).files as $x) | (select(fileIndex == 1).files) += $x' latest-mac-arm64.yml latest-mac-x64.yml | awk '/---/ {p = !p; next} p' > latest-mac.yml
```

I am not sure whether I like us introducing `yq`, which is quite a special tool. :-) Should we maybe comment what this line is supposed to do?

Anyhow, testing it and looking through https://mikefarah.gitbook.io/yq/operators/multiply-merge#merge-flags it seems to me that this also works fine:
```suggestion
                  yq '. *=+ load("latest-mac-x64.yml")' latest-mac-arm64.yml > latest-mac.yml
```